Content/NPCs/Town/Blacksmith.cs

```diff
@@ -65,7 +65,7 @@ public override void HitEffect(NPC.HitInfo hit)
 
 	public override List<string> SetNPCNameList()
 	{
-		return [Language.GetOrRegister("Mods.PathOfTerraria.NPCs.Blacksmith.DisplayName").Value];
+		return [""];
```

Comment:
Was this intended?

Reply:
Yes. This stops the NPC from being called "Name the Name", instead just being called "Name" since it has no "unique name" so to speak. It's a weird interaction.